 Network caique_default  Creating
 Network caique_default  Created
 Container caique-db-1  Creating
 Container caique-db-1  Created
 Container caique-api02-1  Creating
 Container caique-api01-1  Creating
 Container caique-api02-1  Created
 Container caique-api01-1  Created
 Container caique-nginx-1  Creating
 Container caique-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | performing post-bootstrap initialization ... ok
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | waiting for server to start....2024-02-16 05:25:47.383 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-16 05:25:47.386 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-16 05:25:47.395 UTC [51] LOG:  database system was shut down at 2024-02-16 05:25:46 UTC
db-1     | 2024-02-16 05:25:47.423 UTC [48] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/script.sql
db-1     | You are now connected to database "rinha" as user "admin".
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | INSERT 0 5
db-1     | 
db-1     | 
db-1     | waiting for server to shut down...2024-02-16 05:25:47.782 UTC [48] LOG:  received fast shutdown request
db-1     | .2024-02-16 05:25:47.786 UTC [48] LOG:  aborting any active transactions
db-1     | 2024-02-16 05:25:47.788 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db-1     | 2024-02-16 05:25:47.788 UTC [49] LOG:  shutting down
db-1     | 2024-02-16 05:25:47.791 UTC [49] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-16 05:25:48.307 UTC [49] LOG:  checkpoint complete: wrote 940 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.385 s, sync=0.094 s, total=0.519 s; sync files=312, longest=0.064 s, average=0.001 s; distance=4306 kB, estimate=4306 kB; lsn=0/191F590, redo lsn=0/191F590
db-1     | 2024-02-16 05:25:48.311 UTC [48] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-16 05:25:48.406 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-16 05:25:48.406 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-16 05:25:48.406 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-16 05:25:48.412 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-16 05:25:48.435 UTC [67] LOG:  database system was shut down at 2024-02-16 05:25:48 UTC
db-1     | 2024-02-16 05:25:48.442 UTC [1] LOG:  database system is ready to accept connections
nginx-1  | 2024/02/16 05:25:48 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.240.3:8081/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 05:25:48 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.240.4:8082/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 05:25:50 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 05:25:52 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 05:25:54 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 05:25:56 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 05:25:58 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 192.168.240.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://backend/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
api01-1 exited with code 0
api01-1 exited with code 0
api02-1 exited with code 0
api02-1 exited with code 0
db-1     | 2024-02-16 05:30:15.539 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-02-16 05:30:15.544 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-02-16 05:30:15.546 UTC [1] LOG:  background worker "logical replication launcher" (PID 70) exited with exit code 1
db-1     | 2024-02-16 05:30:15.547 UTC [65] LOG:  shutting down
db-1     | 2024-02-16 05:30:15.552 UTC [65] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-16 05:30:15.718 UTC [65] LOG:  checkpoint complete: wrote 786 buffers (4.8%); 0 WAL file(s) added, 0 removed, 2 recycled; write=0.097 s, sync=0.046 s, total=0.172 s; sync files=32, longest=0.023 s, average=0.002 s; distance=25664 kB, estimate=25664 kB; lsn=0/322F898, redo lsn=0/322F898
db-1     | 2024-02-16 05:30:15.722 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
